Makefile

```diff
@@ -1,25 +1,24 @@
 transifex_resource = frontend-app-ecommerce
-langs = "ar,fr,es_419,zh_CN"
-
-transifex_utils = ./src/i18n/i18n-concat.js
+transifex_langs = "ar,fr,es_419,zh_CN"
```

Comment:
Any reason not to include our beta languages?

Reply:
Just scope creep on this task - wasn't part of what I was tryin' to get done... (also none of them have them yet)